### PR TITLE
Fix changelog section ordering 

### PR DESCRIPTION
**Pull request by antchan2**
#### Related Issue: 

N/A

#### Description of changes:

Fix ~~two~~ one minor issue~~s~~ I noticed when reviewing the change-log after the [latest update](https://github.com/ocsf/ocsf-schema/commit/8d86a98f7ea225736e42843e774c21002943d86a):

~~1. The Unreleased Misc. section missed a recently added line: ...~~

2. The "Bugfixes" section somehow got moved inside the "Improved" section, resulting in incorrect indent and formatting.  In 1.3.0, Bugfixes follow Improved Profiles:

image

But in Unreleased, the order was reversed:

image

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -151,6 +150,7 @@ Thankyou! -->
         - Objects; top level attribute allowing link(s) about an object
     - The `source` and `references` attributes are also supported in when extending or patching event classes and objects.
 1. Fixed minor spelling mistakes in attribute descriptions in `dictionary.json`. #1213
+1. Added `http_request` and `http_response` to the evidences object. #1212
```
Comment (floydtree) [inline CHANGELOG.md:153]: This is a duplicate entry, look at line 108. Was fixed earlier, but got brought back in an overriding PR.

**Review by floydtree — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (antchan2) [reply CHANGELOG.md]: Fixed in 05e7fa7

Comment (floydtree) [reply CHANGELOG.md]: thank you!

**Review by floydtree — APPROVED** [review]
Just need to update the PR description to reflect the now-removed changes.

**Comment by antchan2** [comment]
I've updated the PR title and description accordingly.

**Review by jonrau-at-queryai — APPROVED** [review]
Nice 👍

**Review by rmouritzen-splunk — APPROVED** [review]

**Pull request merged**